helsingborg-stad/app-mitt-helsingborg
# fix: show broken-encryption button when needed

## Explain the changes you’ve made

Show CTA button when case encryption is broken.

## Explain why these changes are made

It didn't show during some states.

## Explain your solution

Added extra bool checks to the Card prop.

## How to test

Concrete example:

1. Checkout this branch
2. Run through recurring case with coapplicant until main applicant has signed
3. Erase device data
4. Login and verify it says that something is wrong and the button to fix it is shown and works.

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.

## Screenshots

![image](https://user-images.githubusercontent.com/2890987/208426041-327b7115-c345-4297-8c7d-8fb02dfc5619.png)
